frontend/src/scenes/billing/Billing.tsx

```diff
@@ -237,38 +208,92 @@ export function Billing(): JSX.Element {
+                            <div>
+                                <p className="ml-0 mb-0">
+                                    {billing?.has_active_subscription ? 'Billing period' : 'Cycle'}:{' '}
+                                    <b>{billing.billing_period.current_period_start.format('LL')}</b> to{' '}
+                                    <b>{billing.billing_period.current_period_end.format('LL')}</b> (
+                                    {billing.billing_period.current_period_end.diff(dayjs(), 'days')} days remaining)
+                                </p>
+                                {!billing.has_active_subscription && (
+                                    <p className="italic ml-0 text-muted">
+                                        Monthly free allocation resets at the end of the cycle.
+                                    </p>
+                                )}
+                            </div>
                         </div>
                     </div>
                 )}
 
-                <div
-                    className={clsx('space-y-2', {
-                        'p-4': size === 'medium',
-                    })}
-                    // eslint-disable-next-line react/forbid-dom-props
-                    style={{ width: size === 'medium' ? '20rem' : undefined }}
-                >
-                    {!cloudOrDev && billing?.license?.plan ? (
-                        <div className="bg-primary-alt-highlight text-primary-alt rounded p-2 px-4">
-                            <div className="text-center font-bold">
-                                {capitalizeFirstLetter(billing.license.plan)} license
+                {!isOnboarding && over20kAnnual && (
+                    <div className="flex flex-row gap-8">
+                        <div className="h-30 self-center">
+                            <BuilderHogWavingSmiling className="max-h-full w-auto object-contain" />
+                        </div>
+                        <div className="flex flex-col">
+                            <LemonLabel>You're elgibile for PostHog perks!</LemonLabel>
+                            <LemonCheckbox checked disabled label="Save 20% by switching to up-front annual billing" />
+                            <LemonCheckbox
+                                checked
+                                disabled
+                                label="Discounts for bundling subscriptions to multiple products"
+                            />
+                            <LemonCheckbox checked disabled label="Team training" />
+                            <LemonCheckbox checked disabled label="Dedicated support in a Slack channel" />
+                            <LemonCheckbox checked disabled label="Free merch" />
+                            <div style={{ alignSelf: 'start', paddingTop: '8px' }}>
+                                <LemonButton type="secondary" to="mailto:sales@posthog.com">
+                                    Get in touch!
```

Comment:
We can collapse this once they click the button. Need to think about collapsed state.

Reply:
Is it better to have an explicit dismiss button? Either something that appears on hover of the hog, say, or a x in the upper right that appears on hover?

Reply:
Or a dismiss button next to the Get in touch <<< this is probably the simplest.